tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -699,3 +699,219 @@ describe('Reauthenticating in game', () => {
     cy.get('#deck').should('be.visible');
   });
 });
+
+describe('Reconnecting after game is over', () => {
+  beforeEach(() => {
+    cy.setupGameAsP0();
+  });
+  
+  it('Dialogs persist after refreshing when game is over by conceded and opponent request rematch', () => {
+    cy.concedeOpponent();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.url().then((url) => {
+      const oldGameId = Number(url.split('/').pop());
+      cy.rematchOpponent({ gameId: oldGameId, rematch: true });
+    });
+    cy.get('[data-cy=lobby-ready-card]').should('be.visible');
+    cy.reload();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.get('[data-cy=match-score-counter-wins]').should('contain', 'W: 1');
+    cy.get('[data-cy=lobby-ready-card]').should('exist');
+  });
+
+  it('Dialogs persist after refreshing when game is over by conceded and player request rematch', () => {
+    cy.concedeOpponent();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.get('[data-cy=gameover-rematch]').click();
```

Comment:
Same here, we could reload before clicking rematch just to confirm that it still shows up before rematch is hit